### DISCUSSION TITLE
How do I install query_toml in translate.py ?

### DISCUSSION THREAD

**Discussion by tjyang** [immunant/c2rust#764] [Q&A]
- In libxml2 example when running ./translate.py,  common.py was called up has query_toml
```
from query_toml import query_toml
```
- How do I install it ? I am using rockeylinux 8.
- End

**Comment by thedataking** [comment]
I'm not familiar with Rocky Linux 8 but generally, you can run `python3 -m pip install --user -r scripts/requirements.txt` if you want to make the packages available to all python scripts run by your user account. If you don't want to modify the state of your system, you could use a [Python venv.](https://docs.python.org/3/library/venv.html)
